packages/base/src/base-element.ts

```diff
@@ -0,0 +1,96 @@
+/**
+@license
+Copyright 2018 Google Inc. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import {LitElement} from '@polymer/lit-element';
+import {TemplateResult} from 'lit-html';
+export {TemplateResult};
+export * from '@polymer/lit-element';
+export {classMap} from 'lit-html/directives/classMap';
+export {observer} from './observer';
+
+export interface Adapter {
+  [name: string]: Function
+};
+
+/** Extend this Foundation with the functions you use */
+export interface Foundation {
+  init(): void;
+  destroy(): void;
+}
+
+export declare var Foundation: {
+  prototype: Foundation;
+  new(adapter: Adapter): Foundation;
+}
+
+export abstract class BaseElement extends LitElement {
```

Comment:
So this BaseElement is only supposed to be used by the FormElement and all other elements should inherit directly from LitElement? maybe deserves a comment